src/app/services/indicator.service.ts

```diff
@@ -10,15 +10,16 @@ import { apiHost } from '../constants';
 
 /*
  * Indicator Service
- * Returns climate indicators. Used by the sidebar.
+ * Returns climate indicators. Used by the sidebar and to retrieve detail data for charts.
  */
 @Injectable()
 export class IndicatorService {
     constructor(private apiHttp: ApiHttp) {}
 
     public getData(options: IndicatorQueryOpts) {
-        const url = `${apiHost}/api/climate-data/${options.city.id}/${options.scenario.name}` +
-                  `/indicator/${options.indicator.name}/`;
+
+        const getDetailUrl = `${apiHost}/api/climate-data/${options.city.id}/${options.scenario.name}` +
```

Comment:
Any particular reason for this change? Maybe `dataUrl` instead, since this isn't so much a 'detail' view?